### ISSUE TITLE
Failed when I create a new desktop

### ISSUE THREAD

**Issue by fgonzalezmacias** [isard-vdi/isard#355]
I have an installation update with the develop version to actual date -today-.

When I try to create a new desktop, I obtain an error FailedDomainCreate or similar. Then I try to delete the desktop but it's no successfully

![image](https://user-images.githubusercontent.com/72935746/102696515-4eed4200-422f-11eb-9927-cbc9496e4dbb.png)

I attach the engine-error.log

The hypervisor is online

I have created another desktop and template that work without problems. These were created at least two weeks ago

![image](https://user-images.githubusercontent.com/72935746/102696621-f10d2a00-422f-11eb-9ca7-99e7faac210a.png)

[engine-error.log](https://github.com/isard-vdi/isard/files/5719974/engine-error.log)

**Comment by fgonzalezmacias** [comment]
I just updated my develop version and I was able to remove the desktop already, but when I have gone to create a new one, I get the FailedCreatingDomain error

![image](https://user-images.githubusercontent.com/72935746/102777463-e74f0800-4390-11eb-8526-a7ccf998d973.png)

Any idea? 



Have you tried to create a new desktop from an iso in the current state of the develop version?

Thanks in advance

**Comment by sim6** [comment]
You changed the hypervisor at 2020-12-19 18:06:59, this trigger a restart of engine, but not worked so since that point the engine is failing. You need to restart it manually:
```
docker-compose restart isard-engine
```
Please, could you upgrade with the last develop that includes https://gitlab.com/isard/isardvdi/-/merge_requests/40 that fix the engine restart?

**Comment by fgonzalezmacias** [comment]
Yes I just to update my develop version, and I finally can to delete the desktop, but I want to create another desktop and I obtain the error that I show, FailedCreatingDomain 

I did docker-compose restart isard-engine

![image](https://user-images.githubusercontent.com/72935746/102780681-a823b580-4396-11eb-80aa-82ae9f1c246e.png)


Thanks

**Comment by sim6** [comment]
Thanks. What say the logs?

**Comment by fgonzalezmacias** [comment]
I upload the engine-error.log
[engine-error.log](https://github.com/isard-vdi/isard/files/5724585/engine-error.log)



2020/12/21 13:12:34 136 - ERROR - worker_isard-hypervisor: Exception in libvirt starting paused xml for domain _local-default-admin-admin-W10_Base_2_ in hypervisor isard-hypervisor. Exception message: ('internal error: process exited while connecting to monitor: '
 'qemu-system-x86_64: -blockdev '
 '{"driver":"file","filename":"/isard/media/default/default/local/admin-admin/Windows: '
 'JSON parse error, stray '
 '\'"/isard/media/default/default/local/admin-admin/Windows\'')

**Comment by fgonzalezmacias** [comment]
2020-12-23 18:09:18,360 - changes - INFO - changes_domains - domain _local-default-admin-admin-Prueba_2 deleted from database
libvirt: QEMU Driver error : internal error: process exited while connecting to monitor: qemu-system-x86_64: -blockdev {"driver":"file","filename":"/isard/media/default/default/local/admin-admin/Windows: JSON parse error, stray '"/isard/media/default/default/local/admin-admin/Windows'
2020-12-23 18:10:34,511 - workers - ERROR - worker_isard-hypervisor - Exception in libvirt starting paused xml for domain _local-default-admin-admin-W10_Prueba in hypervisor isard-hypervisor. Exception message: ('internal error: process exited while connecting to monitor: '
 'qemu-system-x86_64: -blockdev '
 '{"driver":"file","filename":"/isard/media/default/default/local/admin-admin/Windows: '
 'JSON parse error, stray '
 '\'"/isard/media/default/default/local/admin-admin/Windows\'') 
2020/12/23 18:10:34 511 - ERROR - worker_isard-hypervisor: Exception in libvirt starting paused xml for domain _local-default-admin-admin-W10_Prueba in hypervisor isard-hypervisor. Exception message: ('internal error: process exited while connecting to monitor: '
 'qemu-system-x86_64: -blockdev '
 '{"driver":"file","filename":"/isard/media/default/default/local/admin-admin/Windows: '
 'JSON parse error, stray '
 '\'"/isard/media/default/default/local/admin-admin/Windows\'') 



This route is the key of the error: /isard/media/default/default/local/admin-admin/Windows:



I think I have found the cause of the possible error, and it is related to some change that had to be made with respect to the Isos (Media) files


If the Windows ISO in the reported directory is saved with some blank space, it is not able to use it later, when the desktop is launched from said ISO

The 'Windows 10' was my original ISO and I haven't problems with it, but with the lasts releases of development any change. Windows10x it's a copy to test my theory. 

With Windows10x no problem, while with my original 'Windows 10' if exist problem to start the desktop


In the latest releases of the develop branch, some change had to be made, that if the ISO name contains any blank space, later when generating the desktop, it is not able to reference it correctly and cuts in the first word


I hope I have explained myself correctly

Thanks in advance


![image](https://user-images.githubusercontent.com/72935746/103028504-309d8400-4558-11eb-9e59-6fc67850828b.png)


![image](https://user-images.githubusercontent.com/72935746/103028560-4e6ae900-4558-11eb-9d9b-442e26b05477.png)
